hs_core/hydroshare/utils.py

```diff
@@ -615,11 +595,11 @@ def convert_file_size_to_unit(size, to_unit, from_unit='B'):
     :param from_unit: should be one of the five: 'B', 'KB', 'MB', 'GB', or 'TB'
     :return: the size converted to the pass-in unit
     """
-    unit = to_unit.lower()
-    if unit not in ('kb', 'mb', 'gb', 'tb'):
-        raise ValidationError('Pass-in unit for file size conversion must be one of KB, MB, GB, '
+    unit = to_unit.lower().replace("i", "")
```

Comment:
Not sure where/how to document this but we decided to keep the GB messaging but use GiB when allocating quota. This is consistent with what we were doing before (the factor below is using 1024 instead of 1000).